frontend/components/App.jsx

```diff
@@ -23,33 +23,11 @@ export default function App() {
         return;
       }
       let location = await Location.getCurrentPositionAsync({});
-      setGeolocation(location);
+      setCoordinates(location);
+      setCoordinates([location.coords.latitude, location.coords.longitude])
```

Comment:
Do you mean to be setting coordinates twice back to back? If so, please let me know. Otherwise, please remove one of the two.